server/.env.template

```diff
@@ -19,3 +19,4 @@ DS__OCA_URI="http://oca.example.org/123/"
 
 DS__ONTOLOGY_URL="https://www.w3.org/ns/dcat.ttl"
 DS__SHACL_URL="https://semiceu.github.io/DCAT-AP/releases/3.0.0/shacl/dcat-ap-SHACL.ttl"
+DS__OCA_BUNDLES_BASE_URL="https://oca-repository.marketplace.nextgen.hiro-develop.nl/oca-bundles"
```

Comment:
please add global as later on we will add the local 
so u can make it DS__global_OCA .....

and update the values.yaml